2020_Fall/readme.md

```diff
@@ -55,7 +55,7 @@ Make the Pull Request from your own GitHub account. Make sure you're changing th
 
 - [ ] Helms, Colton
 
-- [ ] Hirata, Erika
+- [X] Hirata, Erika      kahirata ✨
```

Comment:
Hi Erika, it looks like your Github username is misspelled here. Continue editing on this branch